<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Recipe_input</class>
 <widget class="QDialog" name="Recipe_input">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLineEdit" name="lineEdit_titel"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>16</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>168</width>
           <height>16</height>
          </size>
         </property>
         <property name="sizeIncrement">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="text">
          <string>Kategorie</string>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox">
         <property name="minimumSize">
          <size>
           <width>280</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_add_category">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="myres.qrc">
           <normaloff>:/icons/icons/thin-0151_plus_add_new.png</normaloff>:/icons/icons/thin-0151_plus_add_new.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QListView" name="listView"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QToolButton" name="toolButton_add_ingredient">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="myres.qrc">
             <normaloff>:/icons/icons/thin-0001_compose_write_pencil_new.png</normaloff>:/icons/icons/thin-0001_compose_write_pencil_new.png</iconset>
           </property>
           <property name="shortcut">
            <string>+</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_edit_ingredient">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="myres.qrc">
             <normaloff>:/icons/icons/thin-0002_write_pencil_new_edit.png</normaloff>:/icons/icons/thin-0002_write_pencil_new_edit.png</iconset>
           </property>
           <property name="shortcut">
            <string>Backspace</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_delete_ingredient">
           <property name="text">
            <string>delete</string>
           </property>
           <property name="icon">
            <iconset resource="myres.qrc">
             <normaloff>:/icons/icons/thin-0056_bin_trash_recycle_delete_garbage_empty.png</normaloff>:/icons/icons/thin-0056_bin_trash_recycle_delete_garbage_empty.png</iconset>
           </property>
           <property name="shortcut">
            <string>Del</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="plainTextEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit_titel</tabstop>
  <tabstop>listView</tabstop>
  <tabstop>toolButton_add_ingredient</tabstop>
  <tabstop>toolButton_edit_ingredient</tabstop>
  <tabstop>toolButton_delete_ingredient</tabstop>
  <tabstop>plainTextEdit</tabstop>
 </tabstops>
 <resources>
  <include location="myres.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Recipe_input</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>224</x>
     <y>471</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Recipe_input</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>292</x>
     <y>477</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
